core/common/build.gradle.kts

```
@@ -30,3 +30,7 @@ plugins {
 android {
     namespace = "org.sopt.official.common"
 }
+
+dependencies {
+    implementation(libs.okhttp)
+}
```

```suggestion
dependencies {
    implementation(platform(libs.okhttp.bom))
    implementation(libs.okhttp)
}
```

이렇게 해보셈. 그럼 okhttp 버전 지워도 됨

---

적용 완료!!
platform이라는걸 처음봐서 공부하고 수정하느라 시간이 좀 걸렸습니다 ㅎㅎ